<launch>
    <node pkg="tabletop_pushing" type="tabletop_push_node.py" name="tabletop_push_node" output="screen">
      <param name="collision_aware_ik" type="int" value="0"/>
    </node>
</launch>
